src/Features/Analytics.php

```diff
@@ -163,4 +200,11 @@ public function register_pages() {
 			}
 		}
 	}
+
+	/**
+	 * "Clear" analytics cache by invalidating it.
+	 */
+	public function run_clear_cache_tool() {
+		Cache::invalidate();
```

Comment:
I think we should return a success message (currently empty) after this so it appears after the tool has run.

<img width="486" alt="Screen Shot 2020-06-15 at 8 38 40 PM" src="https://user-images.githubusercontent.com/10561050/84688702-72fe5c80-af48-11ea-8963-ea6ac3f14f30.png">

Reply:
Done in 8278ff49eb7f3be728faec57423249573e5dcf34